util/label/label.rs

```diff
@@ -148,6 +148,7 @@ fn consume_package_name<'s>(input: &'s str, label: &'s str) -> Result<(&'s str,
             || c == '('
             || c == ')'
             || c == '_'
+            || c == '+'
```

Comment:
Can you update the error below too on line 156?

Reply:
I can later today but if you wanted to add apply a suggestion I’d be happy with that too 😅

Reply:
I totally would have, but Github won't let me suggest on a line you didn't edit :(

Reply:
Gah! c'mon github!

-- Updated 😄